src/locationinfo.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// component that has information on the latitude, longitude, county, and town
+class LocationInfo extends React.Component {
+
+  // called when the component is mounted to the DOM
+  componentDidMount() {
```

Comment:
Remember to delete unused functions in your code upon submission of your project.